components/FormStyles/QuestionLayout.tsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components'
 interface QuestionProps {
   questionText: JSX.Element
   input: JSX.Element
+  tooltip?: JSX.Element
```

Comment:
Notes for people reviewing: Ignore this file, these are changes from the other PR